package/MDAnalysis/analysis/density.py

```diff
@@ -170,10 +153,7 @@
 .. _`gmx trjconv`: http://manual.gromacs.org/programs/gmx-trjconv.html
 
 """
-
-from __future__ import print_function, division, absolute_import
-from six.moves import range, zip
-from six import raise_from, string_types
+from __future__ import absolute_import
```

Comment:
Thank you for the explanation why this is still there. I didn’t understand it on the other PR that I looked at. The linter PR is merged so this can go (when you merge develop or rebase).

Reply:
I'm going to open a separate "remove py2 stuff from analysis methods" PR coming along at some point this morning, I'll remove it with that if that works for you?

Reply:
See #2753